apps/storybook/stories/react-essentials/useTimer.stories.tsx

```diff
@@ -37,33 +37,41 @@ export const Default = () => {
   const timer = useTimer(action("callback"), 3000);
 
   return (
-    <Center>
-      <Column gap="1rem" align="center">
-        <TimerProgress {...timer} />
-        <Row gap="1rem">
-          <TimerControls {...timer} />
-          <TimerExtension {...timer} />
-        </Row>
-      </Column>
-    </Center>
+    <Column gap="1rem" align="center">
```

Comment:
Same destructing thingies.xd

Reply:
we discussed that i will destructure timer directly in the story to make it clear what timer contains, and i will pass only the relevant bits to the Timer* components rather than spreading, and then to keep the performance optimal, i might need to make a separate `TimerDebugOverlay` to call the useProgress in there.